tests/framework/installer/install_data.go

```diff
@@ -0,0 +1,266 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package installer
+
+import "strings"
+
+//InstallData wraps rook yaml definitions
+type InstallData struct {
+}
+
+//NewK8sInstallData creates new instance of InstallData struct
+func NewK8sInstallData() *InstallData {
+	return &InstallData{}
+}
+
+func (i *InstallData) getRookOperator(k8sVersion string) string {
+
+	if strings.Contains(k8sVersion, "v1.5") {
+		return `apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rook-operator
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: rook-operator
+    spec:
+      containers:
+      - name: rook-operator
+        image: rook/rook:master
+        args: ["operator"]
+        env:
+        - name: ROOKD_REPO_PREFIX
+          value: roo`
+	}
+
+	return `kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
we were using them from source, but we ran into issues while running tests from the test binaries.